packages/theming/apple-theme/src/appleTheme.typography.ts

```diff
@@ -0,0 +1,55 @@
+import { FontSize, FontSizes, FontWeightValue, Spacing, Typography, Variants } from '@fluentui-react-native/theme-types';
+
+export function appleTypography(): Typography {
+  const appleDict = {
+    sizes: {
```

Comment:
These values should really be pulled from Appkit/UIKit directly. The way it is now, when Apple inevitably tweaks these values, we're going to have to come in here and redo all this work.  I think the best design would be to have a native module that reads them directly from the OS APIs. I implemented [something similar](https://office.visualstudio.com/ISS/_git/sdx-platform?path=%2Fpackages%2Fpolyester%2Fapple%2Fpolyester%2Fmac%2FPLYColorPalette_mac.m&_a=contents&version=GBmaster) in Polyester that seems pretty analogous to what we want here

Reply:
These sizes are not actually coming from Appkit, but from the [Fluent vNext macOS Figma](https://www.figma.com/file/RKqH3UfoIZpO1ZZIlMSp0j/Fluent-vNext-macOS). They are designed to be close to the macOS HIG, but can't quite match it as we haven't moved to a unified type ramp between the different FLURN platforms yet.

Agree that we should pull values like Colors from a native module. for V1 I didn't want to add the extra complexity of that just yet, and I think typography will end up not coming from a native module since they will actually come from design tokens.

Reply:
I do realize I explicitly wrote that they come from the HIG in the description though, ha. I should probably clarify that somewhere

Reply:
Generally speaking I've found it helpful for our consumers to have access to native ideas in addition to what we're (eventually going to be) consuming through the token pipeline, fwiw.  No objection to tackling the problem later though